### ISSUE TITLE
Install from HEAD fails on MacOS Sierra

### ISSUE THREAD

**Issue by jeroen** [grpc/homebrew-grpc#90]
```
Jeroens-MacBook-Pro:~ jeroen$ brew install grpc --HEAD
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
No changes to formulae.

==> Installing grpc from grpc/grpc
==> Cloning https://github.com/grpc/grpc.git
Updating /Users/jeroen/Library/Caches/Homebrew/grpc--git
==> Checking out branch master
==> git clone --branch=nanopb-0.3.5 https://github.com/nanopb/nanopb.git third_party/nanopb
Last 15 lines from /Users/jeroen/Library/Logs/Homebrew/grpc/01.git:
2016-09-28 13:28:38 +0200

git
clone
--branch=nanopb-0.3.5
https://github.com/nanopb/nanopb.git
third_party/nanopb

fatal: destination path 'third_party/nanopb' already exists and is not an empty directory.

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/brew):
  https://github.com/grpc/homebrew-grpc/issues
```

**Comment by muxi** [comment]
The hack in [grpc.rb](https://github.com/grpc/homebrew-grpc/blob/d06ccf5fc39d7cda8e2dafe2ef825f2780c4d520/Formula/grpc.rb#L22-L25]) seems to be   fetching nanopb while grpc's HEAD already has third_party/nanopb in place.

@jtattermusch can you confirm?

**Comment by jtattermusch** [comment]
@muxi  yes, what you are saying makes sense. I think it's a simple fix. @muxi, do you want to look into that?

**Comment by muxi** [comment]
Yea sure.

**Comment by muxi** [comment]
Resolved in #91
